iati/core/default.py

```diff
@@ -139,23 +139,15 @@ def organisation_schemas():
     return output
 
 
-def schemas(bypass_cache=False):
+def schemas():
```

Comment:
The parameter has been removed here, though the functionality still remains.

`_SCHEMAS` exists to act as a cache. Should a Schema already exist in this cache, it would not be read from disk. Due to the implementation of `activity_schemas()` and `organisation_schemas()`, this caching functionality is not used.

In the current state there are statements such as `if (name not in _SCHEMAS.keys()):` which perform checks that are unwanted or harmful.

Either `activity_schemas()` and `organisation_schemas()` should be modified to look in `_SCHEMAS` before reading from disk, or `_SCHEMAS` should be removed.